content/docs/user-guide/what-is-dvc.md

```diff
@@ -1,19 +1,7 @@
 # What Is DVC?
 
-**Data Version Control** is a new type of data versioning, workflow, and
-[experiment management](/doc/user-guide/experiment-management) software, that
-builds upon [Git](https://git-scm.com/) (although it can work stand-alone). DVC
-reduces the gap between established engineering toolsets and data science needs,
-allowing users to take advantage of new [features](#core-features) while reusing
-existing skills and intuition.
-
 ![](/img/reproducibility.png) _DVC codifies data and ML experiments_
 
-Data science experiment sharing and collaboration can be done through a regular
-Git flow (commits, branching, pull requests, etc.), the same way it works for
-software engineers. Using Git and DVC, data science and machine learning teams
```

Comment:
Most of this is already mentioned in the docs home and Exp Mgmt docs so I'm just removing it. However maybe some changes to the **ML experiment management** text in the [home page](https://dvc.org/) could recover some of these ideas (rel https://github.com/iterative/dvc.org/issues/3833).